packages/storybook-vue/src/stories/Link/argTypes.ts

```diff
@@ -0,0 +1,24 @@
+export const argTypes = {
+  external: {
+    name: 'external',
+    type: { name: 'boolean', required: false },
+    defaultValue: false,
+  },
+  href: {
+    name: 'href',
+    type: { name: 'string', required: false },
+    defaultValue: false,
+  },
+  target: {
+    type: 'select',
+    options: ['_self', '_blank', '_parent', '_top'],
```

Comment:
I'm not sure we need to include any `_self`, `_parent` and `_top`. Just empty string and `_blank` would be sufficient.

Including `_parent` and `_top` implies we are okay with using frames and and `<base target="_new">` and I don't think we should be.